config/routes.rb

```diff
@@ -1,22 +1,17 @@
 Rails.application.routes.draw do
+  get '/landing' => 'welcome#show', id: 'landing'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.